src/app/package-json.js

```diff
@@ -143,8 +145,13 @@ const packageJSON = (current, context) => {
         'browserify . -s %s -o dist/%s.js'
       ]),
 
+      'build:js:collapse': scriptify([
+        'bundle-collapser dist/%s.js',
+        '-o dist/%s.min.js'
+      ]),
+
       'build:js:uglify': scriptify([
-        'uglifyjs dist/%s.js',
+        'uglifyjs dist/%s.min.js',
```

Comment:
Should this suffix be changed to `coll` or something instead of `min`? This confused me momentarily.